app/models/placements/schools/placements/build/placement.rb

```diff
@@ -11,7 +11,7 @@
 #
 # Indexes
 #
-#  index_placements_on_provider_id  (provider_id)
+#  index_placements_on_provider_id  (provider_id)a
```

Comment:
What's this "a" doing?

Reply:
Just hanging out 😅 I'll move them along.